src/ui/print.jl

```diff
@@ -189,21 +204,28 @@ function print_stack(io, report, config, wrote_linfos, depth = 1)
         color = RAIL_COLORS[(depth)%N_RAILS+1]
         print_rails(io, depth-1)
         printstyled(io, "┌ "; color)
-        print_frame_sig(io, frame)
+        print_frame_sig(io, frame, config)
         print(io, " ")
         print_frame_loc(io, frame, config, color)
         println(io)
     end
     print_stack(io, report, config, wrote_linfos, depth + 1)
 end
 
-function print_frame_sig(io, frame)
+function print_frame_sig(io, frame, config)
     mi = frame.linfo
     m = mi.def
     if m isa Module
         Base.show_mi(io, mi, #=from_stackframe=#true)
     else
-        Base.StackTraces.show_spec_sig(IOContext(io, :backtrace=>true), m, mi.specTypes)
+        if config.stacktrace_types_limited
```

Comment:
```suggestion
        if config.stacktrace_types_limited && VERSION ≥ v"1.10.0"
```
I just remembered, we probably want to safeguard for julia version 1.10 somewhere (when this depth limited support was added to Base), is here appropriate? I suppose it could alternatively be done upstream

Reply:
The current version of JET is supported on 1.10 (and higher), so we don't need to care about that:)